src/rgw/rgw_bucket.cc

```diff
@@ -765,7 +765,7 @@ static void set_err_msg(std::string *sink, std::string msg)
 }
 
 int RGWBucket::init(RGWRados *storage, RGWBucketAdminOpState& op_state,
-	std::string *err_msg)
+	std::string *err_msg, map<string, bufferlist> *pattrs)
```

Comment:
why pointer to pattrs (and err_msg, for that matter)--could these be reference/& arguments?